.yarnrc.yml

```diff
@@ -2,293 +2,291 @@ compressionLevel: 0
 
 enableGlobalCache: true
 
-logFilters:
-  - code: YN0013
-    level: discard
+preferAggregateCacheInfo: true
```

Comment:
These lines are the only relevant change, everything else is mi IDE reformatting quotes in the `yaml` file.

Docs: https://yarnpkg.com/configuration/yarnrc#preferAggregateCacheInfo